source/services/telegram.js

```diff
@@ -5,7 +5,7 @@
  */
 
 module.exports = {
-    popupUrl: 'https://telegram.me/share/url?url={url}',
+    popupUrl: 'https://telegram.me/share/url?url={url}&text={text}',
```

Comment:
> So, there's no need to make such pr?
Yep, looks like you’re right :-)
I didn’t know how `text` should be passed until you submitted the PR and I checked how it is actually passed though.

I’d suggest to make this PR update `test.html` (which you’ve already done) and add the `Telegram` section into Readme (as specified in the end of https://github.com/ilyabirman/Likely/issues/46).